### ISSUE TITLE
Label for global mute sound button needs changing/adjusting

### ISSUE THREAD

**Issue by terracoda** [phetsims/john-travoltage#300]
Just testing sonfication from other issue and noticed read out for new global mute sound button was, "Sound On Off button, button".

I think the label for this global mute button should be the same as the previous mute sound button, which I believe, was just, "Mute Sound"

The AT actually, provides "button", and also provides whether it is pressed or not pressed or on or off. It is possible an alert was associated with the previous mute sound button. If so, that should remain the same as well.

**Comment by terracoda** [comment]
I assigned to you @jbphet, but please re-assign if you are not the appropriate person. @jessegreenberg or @zepumph would know the code for the previous mute sound button.

**Comment by zepumph** [comment]
How about that @terracoda?

**Comment by zepumph** [comment]
Oops please ignore the above commits.

**Comment by zepumph** [comment]
http://phettest.colorado.edu/john-travoltage/john-travoltage_en.html?brand=phet&ea&accessibility&supportsSound&supportsEnhancedSound

**Comment by terracoda** [comment]
@zepumph, Mute Sound is the correct label, nice, but I do not get any state information from AT (VoiceOver in my case) about the state of the button. 

I just checked Ohm's Law, and the Mute Sound button does indeed have two alerts, though they are not working properly at the moment.

The alert text is:
1. "Sim sound off."
2. "Sim sound on."

So, ideally for the new global Mute Sound button, I should hear alert 1 when I press it the first time to turn sound off, and then alert 2 when I press the button again, re-enabling the sounds.

In Ohm's Law I hear the alerts, but the issue is that the sound keeps flipping back to the on state, so I get both alerts back to back. I will submit an issue about Ohm's Law, but this is likely a common code problem.

**Comment by emily-phet** [comment]
@terracoda The mute sound buttons in all sims besides the latest JT will need to be redone when published next. All sims with sound going forward will have 'mute sound' button in the nav bar (common code in Joist, I would guess).

**Comment by emily-phet** [comment]
I believe JT is in code review now, so there will be at least a few days before it goes to QA. Please coordinate with @jbphet on the timing of this. I believe these updates would just need to be in before the sim goes to QA for dev testing.

**Comment by terracoda** [comment]
In that case, sounds like I can unassign @jessegreenberg as I touch base with @jbphet.

**Comment by terracoda** [comment]
@zepumph, @jbphet says you have "a few days, longer would be an issue for trying to get this sim published in September." It should be going to QA around the end of this week.

**Comment by zepumph** [comment]
This has been completed, and the sound alerts added, @jbphet please reciew.

**Comment by jbphet** [comment]
I don't know much about the utterance queue, but in general the changes look reasonable.  I'll move forward with the release candidate.  Closing.

**Comment by jbphet** [comment]
After further review, I'm thinking I'll leave this open and have it verified during RC.  The QA team might need some help on how to verify.

**Comment by KatieWoe** [comment]
Looks good on all three screen readers.
